mne_bids/write.py

```diff
@@ -978,6 +979,10 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
         also be a string pointing to a .trans file containing the
         transformation matrix. If None, no sidecar JSON file will be written
         for `t1w`
+    deface : str | None
+        The method for defacing: If `smooth` a guassian kernal will be
+        used to smooth the face area, if `erase`, the image values in
+        the face area will be erased. If None, no defacing will be performed.
```

Comment:
You need to tell in docstring that `raw` and `trans` should be provided. Otherwise, the user may feel scolded for no fault of his own :)